sstables/sstables.cc

```diff
@@ -1546,7 +1546,7 @@ sharding_metadata
 create_sharding_metadata(schema_ptr schema, const dht::sharder& sharder, const dht::decorated_key& first_key, const dht::decorated_key& last_key, shard_id shard) {
     auto prange = dht::partition_range::make(dht::ring_position(first_key), dht::ring_position(last_key));
     auto sm = sharding_metadata();
-    auto&& ranges = dht::split_range_to_single_shard(*schema, sharder, prange, shard).get0();
+    auto ranges = dht::split_range_to_single_shard(*schema, sharder, prange, shard).get0();
```

Comment:
How many similar cases do we have that aren't caught? Need to audit all the changes.